de/model.py

```diff
@@ -66,6 +68,7 @@ def random_params(self):
         beta = 1 + (np.diag(self.pert*np.ones(self.n_x)-1))
         neg = np.ones((self.n_x))
         self.beta = np.insert(beta, self.n_x, values=neg, axis=1)
+        # Combine all parameters into p
         self.p = np.concatenate([eps, w.flatten(), alpha])
         return self.p
 
```

Comment:
Is `comparison()` being used? It doesn't look like it—if not it should be deleted.